harness/determined/keras/_tf_keras_trial.py

```diff
@@ -738,6 +738,10 @@ def _launch_evaluate(self) -> Any:
             self.enqueuers.append(enqueuer)
             validation_data = enqueuer.data()
 
+        if isinstance(validation_data, tf.data.Dataset):
+            # Handle validation_steps, which in Keras only applies to tf.data.Datasets.
+            steps = self.context._fit_validation_steps
```

Comment:
question: did you determined that keras does not support validation steps for keras sequences from docs or from reading the source code?

Reply:
From the docs, admittedly.  Though admittedly, from the docs, it also doesn't support sequences for validation_data at all.

Reply:
From the docs it seemed like validation steps would work for sequences, but they could definitely be inaccurate.

Reply:
The docs say about `validation_steps`: `Only relevant if validation_data is provided and is a tf.data dataset`